src/pages/ScholarshipDetailsPage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import scholarships from '../test/scholarshipsTest';
+
+
+function ScholarshipDetailsPage({match}){
+    //will get replaced once we make actual api calls
+    const scholarshipInfo = scholarships.filter(item => item.id === match.params.id);
+
+    return(
+        <div>
+            <h1>{scholarshipInfo[0].name}</h1>
+            <h2>${scholarshipInfo[0].amount}</h2>
+            <h3>{scholarshipInfo[0].deadline}</h3>
+            <h3>{scholarshipInfo[0].website}</h3>
+            <h3>{scholarshipInfo[0].school}</h3>
+            <h3>{scholarshipInfo[0].year}</h3>
+            <p>{scholarshipInfo[0].description}</p>
+        </div>
+    );
```

Comment:
A few suggestions:
1. Handle the case where the id isn't found, it should be something like 'scholarship not found'
2. Extract the 0th element, `scholarshipInfo` instead of `scholarshipInfo[0]`
3. Nit: rename `scholarshipInfo` to `scholarship`. The `Info` suffix does not clarify the variable's purpose much